extensions/datastores/accumulo/src/main/java/mil/nga/giat/geowave/datastore/accumulo/AccumuloDataStore.java

```diff
@@ -99,6 +115,9 @@
 
 	private final AccumuloSplitsProvider splitsProvider = new AccumuloSplitsProvider();
 
+	private final HashMap<ByteArrayId, Integer> dupCountMap = new HashMap<>();
```

Comment:
these maps cannot be used per datastore - delete() should be able to be called concurrently with the same instance of a datastore (no shared state).